docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,69 @@
+# Troubleshooting guide
+
+## WMCO does not go to running
+Please check if you are using an OKD/OCP 4.6 cluster running on Azure or AWS, configured with
+[hybrid OVN Kubernetes networking](docs/setup-hybrid-OVNKubernetes-cluster.md).
+
+## Windows Machine does not become a worker node
+There could various reasons as to why a Windows Machine does not become a worker node. Please collect the WMCO logs
+by executing:
+```shell script
+# oc logs $(oc get pods -o jsonpath={.items[0].metadata.name}) -n openshift-windows-machine-config-operator
```

Comment:
should we remove the # prefix in the commands, it just creates a lighter font in the rich diff.